geopandas/tests/test_sindex.py

```diff
@@ -371,11 +379,30 @@ def test_intersection_invalid_bounds_tuple(self, test_geom):
             ),  # contains but does not contains_properly
         ),
     )
-    def test_query(self, predicate, test_geom, expected):
+    def test_query(self, predicate, test_geom, expected, output_format):
         """Tests the `query` method with valid inputs and valid predicates."""
         res = self.df.sindex.query(test_geom, predicate=predicate)
         assert_array_equal(res, expected)
 
+        if output_format != "indices":
+            dense = np.zeros(len(self.df), dtype=bool)
+            dense[expected] = True
```

Comment:
```suggestion
        if output_format != "indices":
            dense = np.zeros(len(self.df), dtype=bool)
            dense[expected] = True
```
I think it makes sense to indent the following cases given they're subsets of !=indices

(we also don't test explicitly on passing output_format="indices", but given we do test on omiting output_format and that is the default that seems okay)